src/components/Accordion/Accordion.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+import AccordionSummary from './AccordionSummary'
+import AccordionWrapper from './AccordionWrapper'
+import AccordionDetails from './AccordionDetails'
+
+const Accordion = ({ content, isOpenByDefault, header }) => {
+  const [expanded, setExpanded] = useState(isOpenByDefault ? 'panel1' : '')
```

Comment:
Would it be possible to just use a boolean toggle `isExpanded` here instead? The `'panel1'` value is confusing

Reply:
panel1 is just some self identifier

isOpenByDefault = is indicating that this will only work when it's mounted. So if the value changes it doesn't do anything. `isExpanded` would indicated that this value will change the state

Reply:
To be more particular this is what I mean:

```suggestion
  const [isExpanded, setIsExpanded] = useState(isOpenByDefault)
```

I don't see why the state that determines if the Accordion is open can't just be a `boolean`. I don't see why the `'panel1'` value is necessary

Reply:
you are correct, let me change it